.github/workflows/workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: Run unit tests
+on: ["push", "pull_request"]
+
+jobs:
+  test:
+    name: Dub tests
+    strategy:
+      matrix:
+        os: [macOS-latest, ubuntu-latest, windows-latest]
+        dc: [dmd-latest, ldc-latest]
+        exclude:
+          - { os: macOS-latest, dc: dmd-latest }
```

Comment:
Why you are excluding this?

Reply:
It was on the template I used.

Reply:
Try to remove it. You can also add `-master` branch to prevent issues with future releases. You may disable or mark `master` branch as not required in Github settings.